assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -223,6 +223,23 @@ function APIProvider({ children }) {
     return apiFetch({ path: addQueryArgs(users, { per_page: '-1' }) });
   }, [users]);
 
+  const getMenuFonts = useCallback((fontList) => {
```

Comment:
This is not a call to the api, so should not be here. There is a font provider already...